pandas/tests/generic/test_generic.py

```diff
@@ -948,3 +948,18 @@ def test_deprecated_get_dtype_counts(self):
         df = DataFrame([1])
         with tm.assert_produces_warning(FutureWarning):
             df.get_dtype_counts()
+
+    def test_boolean_in_replace(self):
+        # GH 27660
+        df = DataFrame({"col": [False, True, 0, 1]})
+
+#        result = df.replace({"col": {False: 0, True: 1}})
```

Comment:
Typically better to delete than to comment out

Reply:
wooow... I am so sorry... i didn't intend to push, I am still working on it.. 

when i was doing `git checkout`, it says the branch is one commit ahead, so have to do `git push`... I am really sorry, I will ping when the PR is ready for review. @WillAyd